### PR TITLE
[OYPD-496] Update default color scheme.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

# Jira issue: 
https://propeople-us.atlassian.net/browse/OYPD-496
Drupal.org issue: https://www.drupal.org/node/2886628

# Steps to review:
- [ ] go to frontpage
- [ ] compare color style in the build and http://openy-dev.ffwua.com/public/admin/appearance/settings/openy_rose
- [ ] verify build have same style as openy-dev site

### CODE REVIEW

File: themes/openy_themes/openy_rose/color/color.inc [hunk 1]
```diff
@@ -41,7 +41,41 @@ $info = array(
...
+        'campfooter' => '#006b6b',
+        'campfootertext' => '#fffff7',
+        'camptext' => '#636465',
+        'camplink' => '#006b6b',
+        'campbutton' => '#006b6b',
+        'campbuttonlink' => '#fffff9',
+      ),
+    ),
+    'bluesky' => array(
+      'title' => t('Blue sky'),
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/color/color.inc]: This should be Seattle.

File: themes/openy_themes/openy_rose/color/color.inc [hunk 2]
```diff
@@ -41,7 +41,41 @@ $info = array(
   // Pre-defined color schemes.
   'schemes' => array(
     'default' => array(
-      'title' => t('Blue sky (default)'),
+      'title' => t('Seattle (default)'),
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/color/color.inc]: This should be Blue Sky.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1581 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1581 |
| Installation process | http://install.openy-dev.ffwua.com/build1581/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1581/

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1583 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1583 |
| Installation process | http://install.openy-dev.ffwua.com/build1583/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1583/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1583/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1585 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1585 |
| Installation process | http://install.openy-dev.ffwua.com/build1585/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1585/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1585/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1596 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1596 |
| Installation process | http://install.openy-dev.ffwua.com/build1596/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1596/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1596/profiles/contrib/openy/build/reports/behat

**Comment by alexschedrov** [comment]
Some colors are different. Make sure you've updated demo content to use appropriate colors.
image
image

**Comment by ivan-berezhnov** [comment]
@Sanchiz https://github.com/ymcatwincities/openy/pull/568#issuecomment-308952692
Yeah, I saw it but I thought it fix subtask. Now I fixed it

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1606 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1606 |
| Installation process | http://install.openy-dev.ffwua.com/build1606/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1606/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1606/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1607 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1607 |
| Installation process | http://install.openy-dev.ffwua.com/build1607/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1607/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1607/profiles/contrib/openy/build/reports/behat

**Comment by ivan-berezhnov** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1622 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1622 |
| Installation process | http://install.openy-dev.ffwua.com/build1622/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1622/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1622/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]

**Pull request merged**